src/modules/whats-new/components/whats-new-modal.tsx

```diff
@@ -57,6 +57,14 @@ const PageContent = ( {
 export default function WhatsNewModal( { showModal, onClose }: WhatsNewModalProps ) {
 	const locale = useI18nLocale();
 	const whatsNewPages: WhatsNewPage[] = [
+		{
+			image: selectiveSyncIllustration,
+			title: __( 'Choose what to synchronize' ),
+			description: __(
+				'Choose which files and folders to synchronize, speeding up the process and limiting changes to part of your site.'
+			),
```

Comment:
What do you think about this title and description. I tweaked yours a bit.

```suggestion
			title: __( 'Synchronize with precision' ),
			description: __(
				'Choose the parts of your site to sync, speeding up the process and reducing unnecessary changes.'
			),
```

Reply:
Thanks @sejas for proposing the alternative. It also looks good to me. I will update it.